src/ceph-volume/ceph_volume/devices/lvm/listing.py

```diff
@@ -292,17 +226,13 @@ def main(self):
         )
 
         parser.add_argument(
-            'device',
-            metavar='DEVICE',
-            nargs='?',
+            'device', metavar='DEVICE', nargs='?',
```

Comment:
Please revert this change for readability.

Reply:
Reverted.